configuration-example.js

```diff
@@ -1,6 +1,12 @@
 const EluvioConfiguration = {
-  "config-url": "https://main.localhost:8008/config",
-  "apps": {
-    "fabric-browser": "http://localhost:8080"
-  }
-};
+  // Fabric Network
+  "config-url": "https://test.net955203.contentfabric.io/config",
+  // "config-url": "https://main.net955305.contentfabric.io/config",
+  // "config-url": "https://demov3.net955210.contentfabric.io/config",
+
+  //v4
+  // "config-url": "https://host-468.contentfabric.io/config",
+
+  // Fabric Browser
+  "coreUrl": "http://localhost:8082"
```

Comment:
core doesn't need `coreUrl`, it needs `apps`

See https://core.v3.contentfabric.io/configuration.js for defaults.

You can specify that these apps can be replaced with local instances (e.g. fabric browser: "http://localhost:8080")